berkeley-mobile/StudyPact/Profile/ProfileViewController.swift

```diff
@@ -186,9 +186,11 @@ extension ProfileViewController {
     }
     
     @objc private func logoutButtonPressed(sender: UIButton) {
-        let alertController = UIAlertController(title: "Logout?", message: "You will be returned to the login screen.", preferredStyle: .alert)
-        alertController.addAction(UIAlertAction(title: "Cancel", style: UIAlertAction.Style.cancel, handler: nil))
-        alertController.addAction(UIAlertAction(title: "Logout", style: UIAlertAction.Style.destructive, handler: {_ in
+        
+        
+        let alertView = AlertView(headingText: "Logout?", messageText: "You will be returned to the login screen.", action1Label: "Cancel", action1Color: Color.ActionButton.background, action1Completion: {
+            self.dismiss(animated: true, completion: nil)
+        }, action2Label: "Logout", action2Color: Color.AlertView.secondaryButton, action2Completion: {
```

Comment:
looks like the cancel button is blue here, whereas in other places the cancel button is gray.
I think we should make the cancel button gray here and keep that pretty consistent